packages/scenes/src/components/VizPanel/VizPanelRenderer.tsx

```diff
@@ -172,6 +178,68 @@ export function VizPanelRenderer({ model }: SceneComponentProps<VizPanel>) {
           >
             {(innerWidth, innerHeight) => (
               <>
+                {plugin.meta.id === 'timeseries' && (
```

Comment:
not reviewing this assuming this is copied as is from the current impl that works.